config.json

```diff
@@ -1240,7 +1744,12 @@
         "slug": "parallel-letter-frequency",
         "name": "Parallel Letter Frequency",
         "uuid": "1992eaf7-7fbe-4176-ad46-01e325c27bcb",
-        "prerequisites": [],
+        "prerequisites": [
+          "strings",
+          "regular-expressions",
+          "enum",
+          "task"
```

Comment:
The tests don't check this in any way, but I am pretty sure that `Task` is necessary to follow the exercise's instructions ("using parallel computation"). This is not a concept yet, but I feel like it should be. What do you think?

Reply:
https://github.com/exercism/elixir/issues/587